src/bootstrap/server-bootstrapper.js

```diff
@@ -1,13 +1,15 @@
 import url from 'node:url';
 import path from 'node:path';
-import { promises as fs } from 'node:fs';
 import Cdn from '../repositories/cdn.js';
 import Logger from '../common/logger.js';
 import { Container } from '../common/di.js';
 import Database from '../stores/database.js';
 import ServerConfig from './server-config.js';
 import ClientConfig from './client-config.js';
 import PageResolver from '../domain/page-resolver.js';
+import { promises as fs, readFileSync } from 'node:fs';
```

Comment:
For the other file reading matters we use the asynchronous version in this module (`fs.readFile`), I'd propose to not use the blocking `...Sync` versions.